rpcperms/interceptor.go

```diff
@@ -90,16 +158,16 @@ func (r *RPCInterceptor) Permissions() map[string][]bakery.Op {
 // CreateServerOpts creates the GRPC server options that can be added to a GRPC
 // server in order to add this RPCInterceptor.
 func (r *RPCInterceptor) CreateServerOpts() []grpc.ServerOption {
-	macUnaryInterceptors := []grpc.UnaryServerInterceptor{}
-	macStrmInterceptors := []grpc.StreamServerInterceptor{}
+	var unaryInterceptors []grpc.UnaryServerInterceptor
```

Comment:
I think this whole section could be simplified even more by just adding all the interceptors to this slice as a static initialization.

Reply:
I actually like it this way, easier to see that we add them one by one?